assets/js/translator.js

```diff
@@ -29,6 +29,35 @@ var PUNCTUATION_KEY_CODES = [46, 33, 58, 63, 47, 45, 190, 171, 49]; // eslint-di
     callApy, apyRequestTimeout, isURL */
 /* global ENTER_KEY_CODE, HTTP_BAD_REQUEST_CODE, HTTP_OK_CODE, SPACE_KEY_CODE, XHR_DONE, XHR_LOADING */
 
+function updateDetect(state /*: bool */) {
```

Comment:
Could you move this function to a more appropriate location?

Reply:
In terms of module or what?

Reply:
What would be the alternative?

Reply:
Somewhere with the other functions? i.e. around L1026